### ISSUE TITLE
No output at last...I trained the network but on test dataset I am not getting any mask. all file is blank

### ISSUE THREAD

**Issue by SankBad** [jocicmarko/ultrasound-nerve-segmentation#75]

**Comment by KeepHappy-365** [comment]
I encountered the same situation.  And I can't solve it now....

**Comment by washeem** [comment]
I have the same issue as well...

**Comment by SankBad** [comment]
My data was imbalance and when I used appropriate loss function. I got correct output.

**Comment by SoulGuedria** [comment]
HI @SanBadhe,

What was the loss function you used to solve the problem? did you use data augmentation to solve the data imbalance problem ?
Thank you for you help :)

**Comment by SankBad** [comment]
I used loss function as an addition of binary cross entropy and dice coefficient. 
I used data augmentation because of the dearth of data. you can't balance data in image context.

**Comment by SoulGuedria** [comment]
Thanks a lot @SanBadhe :)
